### ISSUE TITLE
Documentation missing in NuGet package

### ISSUE THREAD

**Issue by bender2k14** [louthy/language-ext#436]
When I use the "Go To Definition (F12)" feature in Visual Studio on a type or method in Language Ext, I am taken to file that does not have any documentation.  It only has the signatures.  See this image.

![screenshot 2018-06-13 16 25 30](https://user-images.githubusercontent.com/34664007/41379120-c5707d68-6f26-11e8-87b3-27117e1c40a2.png)

Do other people experience this as well or is my development environment not configured correctly?

**Comment by louthy** [comment]
I think it’s dotnet pack that’s the problem, before that we had docs, I
keep meaning to look into it but very short of time atm

Weirdly VS has started stepping into the source for me now when debugging -
I think it was the latest update, which is super annoying

**Comment by bender2k14** [comment]
Is your build and deployment processes available? If so, then one of us might be able to fix that fire you.

**Comment by louthy** [comment]
After incrementing the version numbers using [`inc.bat`](https://github.com/louthy/language-ext/blob/master/inc.bat) I run [`pack.bat`](https://github.com/louthy/language-ext/blob/master/pack.bat) to build the project:
```c#
dotnet restore
dotnet pack LanguageExt.Core -c Release -o ../../artifacts/bin
dotnet pack LanguageExt.FSharp -c Release -o ../../artifacts/bin
dotnet pack LanguageExt.Parsec -c Release -o ../../artifacts/bin
dotnet pack LanguageExt.Rx -c Release -o ../../artifacts/bin
```
And then run a little console app I wrote to push to nu-get.

It has a function in it for invoking the `nuget.exe`

```c#
        static void Nuget(params string[] parms)
        {
            var process = new Process {
                StartInfo = new ProcessStartInfo {
                    FileName = "nuget.exe",
                    Arguments = String.Join(" ", parms.Select(p => p.Contains(' ') ? $"\"{p}\"" : p)),
                    UseShellExecute = false,
                    RedirectStandardOutput = true,
                    RedirectStandardError = true,
                    CreateNoWindow = true
                }
            };

            process.Start();
            while (!process.StandardOutput.EndOfStream)
            {
                Console.WriteLine(process.StandardOutput.ReadLine());
            }
            while (!process.StandardError.EndOfStream)
            {
                Console.WriteLine(process.StandardError.ReadLine());
            }
        }
```
Which is called like so, for each `nupkg`:

```c#
    Nuget("push", file, apiKey, "-src", "nuget.org");
```
For a while dotnet pack was terrible and refused to work with various aspects of the project files, but I did before all the .NETCore debacle have a source documentation build, but like I say I'm not really time wealthy at the moment, so this has slipped off the radar.

**Comment by louthy** [comment]
This is fixed in `v3.0.23` thanks @rantri
